ipn/ipnlocal/cert.go

```diff
@@ -101,7 +102,7 @@ func (b *LocalBackend) GetCertPEM(ctx context.Context, domain string) (*TLSCertK
 	}
 
 	if pair, err := getCertPEMCached(cs, domain, now); err == nil {
-		shouldRenew, err := shouldStartDomainRenewal(domain, now, pair)
+		shouldRenew, err := b.shouldStartDomainRenewal(cs, domain, now, pair)
 		if err != nil {
 			logf("error checking for certificate renewal: %v", err)
```

Comment:
%w here too? (I don't know; legitimately wondering)

Reply:
Unfortunately `%w` is only supported in `fmt.Errorf` and not in logging packages.
It would be nice if it treated `%w` like `%v`, but it does not :)